packages/skeleton-loader/README.md

```diff
@@ -0,0 +1,63 @@
+# Skeleton Loader
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/skeleton.svg)
+
+#### [View on MongoDB.design](https://www.mongodb.design/component/skeleton/example/)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/skeleton-loader
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/skeleton-loader
+```
+
+## Example
+
+```jsx
+<ParagraphSkeleton />
+<ParagraphSkeleton withHeader />
+<FormSkeleton />
+<CardSkeleton />
+<TableSkeleton columnLabels={['Column 1', 'Column 2']} />
+```
+
+## Properties
+
+#### Skeleton
+
+| Name             | Type      | Default | Description                           |
+| ---------------- | --------- | ------- | ------------------------------------- |
+| `size`           | `boolean` | `false` | Determines the height of the skeleton |
```

Comment:
Don't forgot to change the `Type` and `Default` values

Reply:
yikes thank you